index.html

```diff
@@ -274,6 +275,43 @@ <h1 class="underlined">About Me</h1>
                 </ul>
             </div>
         </section>
+
+        <!-- contact me  -->
+        <section class="get-in-touch">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="https://formspree.io/f/xyyawvzk" enctype="multipart/form-data" method="POST"
+                class="get-in-touch-form">
+                <div class="name-input">
+                    <input type="text" name="first name" placeholder="First Name" maxlength="30" required>
+                    <input class="last-name-input" type="text" name="last name" placeholder="Last Name" maxlength="30"
+                        required>
+                </div><input type="email" name="email" placeholder="Email Address" required>
+                <textarea name="message" id="message" cols="30" rows="5" maxlength="500" required
+                    placeholder="Enter text here">
+                </textarea>
+                <button type="submit">Get in touch</button>
+            </form>
```

Comment:
- [ ] Please make sure your contact form is submitted correctly. Currently, when clicking on the **submit button** nothing happens. This is because there is an unused input field which is:
`<input class="last-name-input" type="text" name="last name" placeholder="Last Name" maxlength="30" required>`
Please remove this inputs field and make sure your form is submitted correctly according to project requirements. 

This will users to contact your without any effort 😄

Reply:
Resolved Required Changes

- Removed the blank line and spaces inside the <textarea>.

- Button interaction like hover added

- Already contact form submission is working perfectly when you fill out all forms and then click on submit button it works perfectly.

- Removed commented line of code